.azure-pipelines/scripts/sap_hana/linux/55_docker_login.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -euo pipefail
+
+set +x
+
+echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin
```

Comment:
> Important: By default with GitHub repositories, secret variables associated with your pipeline are not made available to pull request builds of forks. See Validate contributions from forks.

Does that mean the build will fail for forks ?

Reply:
Yes